src/appleseed/renderer/modeling/project/projectfilewriter.cpp

```diff
@@ -863,7 +863,9 @@ namespace
             write_collection(shader.shader_params());
 
             if (shader.get_source_code())
+            {
```

Comment:
We prefer not to add braces for single line statements in order to increase vertical code density.